### ISSUE TITLE
Committed to the WindowsPhone market have a problem

### ISSUE THREAD

**Issue by AranWang** [danielcrenna/tweetsharp#175]
I submitted to the app store have these questions:
•	Error Found: The debug configuration test detected the following errors:  
o	The binary Hammock.WindowsPhone.dll is built in debug mode.
o	The binary ICSharpCode.SharpZipLib.WindowsPhone.dll is built in debug mode.
o	The binary TweetSharp.WindowsPhone.dll is built in debug mode.

I try to clone code into my computer, but the code to compile.
Please provide a release version.

**Comment by Yortw** [comment]
This repository isn't being actively maintained. If you can get the code to compile you should be able to make and use your own assemblies. If you want release binaries provided for you, please try one of the alternative nuget packages. I believe there is one called Nuget-Unofficial, another one called TweetSharp.WindowsPhone and I have one called TweetMoaSharp. All of these should have working release binaries.

**Comment by AranWang** [comment]
What you said is this?
->https://www.nuget.org/packages/TweetSharp.WindowsPhone/

I use in the project "NuGet" provides a DLL, however, when submitted to the app store, detect these DLL is built in debug mode.

**Comment by Yortw** [comment]
Yes, that is one of the packages I mentioned, you could try that. I haven't used it myself, but it should work. If it doesn't work, you could try TweetMoaSharp, it should also support Windows Phone and should have release binaries. 

Either should work, but if you want to stick with the TweetSharp.WindowsPhone package and it doesn't have release binaries, then you will need to contact the author of that package directly.

**Comment by AranWang** [comment]
Thank you for your answer very much, I will try to contact the author.

**Comment by Yortw** [comment]
Good luck. If you have issues, try switching to one of the other packages. They are all based on the original TweetSharp, so you should be able to switch packages without changing your code.
